ptgnn/implementations/typilus/train.py

```diff
@@ -149,9 +150,12 @@ def run(arguments):
 
     initialize_metadata = True
     restore_path = arguments.get("--restore-path", None)
-    if restore_path or (arguments["--aml"] and model_path.exists()):
+    if restore_path:
         initialize_metadata = False
         model, nn = Graph2Class.restore_model(Path(restore_path))
+    elif arguments["--aml"] and model_path.exists():
```

Comment:
unrelated change?

Reply:
yes, again (allows pre-empted workloads to resume)